.github/workflows/release.yaml

```diff
@@ -0,0 +1,36 @@
+name: Release Cloudformation
+on:
+  workflow_dispatch:
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Setup AWS credentials
+        uses: aws-actions/configure-aws-credentials@v1
```

Comment:
```suggestion
        uses: aws-actions/configure-aws-credentials@v2
```

Outdated

Reply:
resolved in [e93dc8f](https://github.com/observeinc/cloudformation-aws-collection/pull/35/commits/e93dc8f8fbfd74a79452f2fa9adadc7dadd41c64)